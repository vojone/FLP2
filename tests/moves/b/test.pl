:- consult("../../../cube").

:- read_cube(C, _), move_b_cw(C, _, _, RC), write_cube(RC).
