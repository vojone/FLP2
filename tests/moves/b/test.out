122
456
789
444 111 741 333
111 222 852 244
111 223 963 144
666
666
344
